changelog.d/13843.misc

```diff
@@ -0,0 +1 @@
+Remove the `complete_sso_login` method from the Module API which was deprecated in Synapse 1.13.0.
```

Comment:
This should be a `.removal` as per https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#changelog